.github/workflows/staging.yml

```diff
@@ -1,12 +1,52 @@
-name: Staging Deployment
+name: Deploy Staging
 
-on: workflow_dispatch
+on:
+  workflow_dispatch:
+    inputs:
+      deployMain:
+        description: "Deploy main? true/false"
+        required: false
+        default: "true"
 
 jobs:
-  terraform_plan:
-    name: Terraform plan
+  save_docker_image_tags_staging:
+    if: github.event.inputs.deployMain == 'true'
+    name: Copy and save image tags for staging environment
```

Comment:
To be consistent with what we set this to in the `dev` pipeline could we update this to: `update_staging_docker_images`?

See my comment here: https://github.com/mcagov/beacons-integration/pull/25#discussion_r595347786